src/Sylius/Behat/Context/Ui/CartContext.php

```diff
@@ -68,6 +69,7 @@ public function iAddProductToTheCartSelectingVariant($variant, ProductInterface
 
     /**
      * @Given I remove product :productName from the cart
+     * @Given I removed product :productName from the cart
```

Comment:
Can we merge this with the previous? `@Given I (remove|removed) product :productName from the cart` WDYT?